packages/utils/src/lib/report-to-md.ts

```diff
@@ -87,14 +95,41 @@ function reportToCategoriesSection(report: ScoredReport): string {
     )} Score:  ${style(formatReportScore(category.score))}`;
     const categoryDocs = getDocsAndDescription(category);
 
-    const refs = category.refs.reduce((acc, ref) => {
-      if (ref.type === 'group') {
-        acc += groupRefItemToCategorySection(ref.slug, ref.plugin, plugins);
-      } else {
-        acc += auditRefItemToCategorySection(ref.slug, ref.plugin, plugins);
-      }
-      return acc;
-    }, '');
+    const auditsAndGroups = category.refs.reduce(
+      (
+        acc: {
+          audits: WeighedAuditReport[];
+          groups: EnrichedScoredAuditGroupWithAudits[];
+        },
+        ref: CategoryRef,
+      ) => {
+        if (ref.type === 'group') {
+          acc.groups = [
+            ...acc.groups,
+            getGroupWithAudits(ref.slug, ref.plugin, plugins),
+          ];
+        } else {
+          acc.audits = [...acc.audits, getAuditByRef(ref, plugins)];
+        }
+        return { ...acc };
+      },
```

Comment:
You don't need to mutate the accumulator here, use a conditional object spread instead:

```suggestion
      ) => ({
        ...acc,
        ...ref.type === 'group' ? {
          groups: [...acc.groups, getGroupWithAudits(ref.slug, ref.plugin, plugins)],
        } : {
          audits: [...acc.audits, getAuditByRef(ref, plugins)],
        },
      }),
```

Reply:
Fixed